### ISSUE TITLE
[Feature request] Custom notification sounds

### ISSUE THREAD

**Issue by ghost** [qTox/qTox#2639]
Being able to use custom notification sounds would be really great. I know some people who don't like the message received effect at all, and I myself would love to get to use the old MSN notifications.

**Comment by Zer0-One** [comment]
We're going to be adding a standardized naming scheme to the themes in https://github.com/Tox/Sounds, and if qTox picks up support for it, then it should be easier to plug different collections of sounds in and out.

**Comment by tux3** [comment]
I plan to add support when the naming scheme is standardized.

**Comment by ghost** [comment]
So qTox is gonna have themes like MSN had? Cool!

**Comment by tux3** [comment]
We'll at least have sound packs, full themes maybe later.

**Comment by ghost** [comment]
Why can't this be implemented before the standardization? Wouldn't it just require some simple changes to make it compatible with a later standard?

**Comment by zetok** [comment]
> Why can't this be implemented before the standardization?

And who would do the implementation?

**Comment by RowenStipe** [comment]
@zetok It's something that I can work on a long with the Notification Management System I've been working on when I can.

**Comment by ghost** [comment]
Is there any update on this?

**Comment by anthonybilinski** [comment]
No, unfortunately not. Focus has been elsewhere, but this still might happen some day™.
